model/SyncLog/Storage/RdsSyncLogStorage.php

```diff
@@ -0,0 +1,253 @@
+         }
+
+         return $this->persistence;
+     }
+
+     /**
+      * @return \Doctrine\DBAL\Query\QueryBuilder
+      * @throws \oat\oatbox\service\exception\InvalidServiceManagerException
+      */
+     private function getQueryBuilder()
+     {
+         return $this->getPersistence()->getPlatForm()->getQueryBuilder()->from(self::TABLE_NAME);
+     }
+
+     /**
+      * Store SyncLogEntity in rds storage.
+      *
+      * @param SyncLogEntity $entity
+      * @return mixed|void
+      */
+     public function create(SyncLogEntity $entity)
+     {
+         $this->getPersistence()->insert(
+             self::TABLE_NAME,
+             [
+                 SyncLogStorageInterface::COLUMN_SYNC_ID => $entity->getSyncId(),
+                 SyncLogStorageInterface::COLUMN_BOX_ID => $entity->getBoxId(),
+                 SyncLogStorageInterface::COLUMN_ORGANIZATION_ID => $entity->getOrganizationId(),
+                 SyncLogStorageInterface::COLUMN_DATA => json_encode($entity->getData()),
+                 SyncLogStorageInterface::COLUMN_STATUS => $entity->getStatus(),
+                 SyncLogStorageInterface::COLUMN_REPORT => json_encode($entity->getReport()),
+                 SyncLogStorageInterface::COLUMN_STARTED_AT => $entity->getStartTime()->format(SyncLogEntity::DATE_TIME_FORMAT)
+             ]
+         );
+     }
+
+     /**
+      * Update synchronization log record.
+      *
+      * @param SyncLogEntity $entity
+      * @return mixed
+      */
+     public function update(SyncLogEntity $entity)
+     {
+         $qb = $this->getQueryBuilder();
+         $qb->update(self::TABLE_NAME)
+             ->set(SyncLogStorageInterface::COLUMN_STATUS, $qb->createNamedParameter($entity->getStatus()))
+             ->set(SyncLogStorageInterface::COLUMN_DATA, $qb->createNamedParameter(json_encode($entity->getData())))
+             ->set(SyncLogStorageInterface::COLUMN_REPORT, $qb->createNamedParameter(json_encode($entity->getReport())));
+
+         $finishTime = $entity->getFinishTime();
+         if ($finishTime instanceof \DateTime) {
+             $finishedAt = $entity->getFinishTime()->format(SyncLogEntity::DATE_TIME_FORMAT);
+             $qb->set(SyncLogStorageInterface::COLUMN_FINISHED_AT, $qb->createNamedParameter($finishedAt));
+         }
+
+         $qb->where(SyncLogStorageInterface::COLUMN_ID . ' = ' . $qb->createNamedParameter($entity->getId()));
+
+         return $this->getPersistence()->exec($qb->getSQL(), $qb->getParameters());
+     }
+
+     /**
+      * @param $id
+      * @return SyncLogEntity
+      * @throws common_exception_NotFound
+      * @throws InvalidServiceManagerException
+      */
+     public function getById($id)
+     {
+         $queryBuilder = $this->getQueryBuilder();
+         $queryBuilder->select('*')
+             ->where(SyncLogStorageInterface::COLUMN_ID . ' = ' . $queryBuilder->createNamedParameter($id, \PDO::PARAM_INT));
+
+         $sql = $queryBuilder->getSQL();
```

Comment:
Why do you need to get `$sql` query string and `$params` array for each request?